application/routes.py

```diff
@@ -0,0 +1,19 @@
+from flask import render_template
+from application import app
+
+
+@app.route('/')
+@app.route('/index')
+def index():
+    to_dos = [
+        {'number': 1,
+         'task_name': 'Открыть FLASK',
+         'status': 'Done'
+         },
```

Comment:
we don't want to have this data defined in the file called **routes.py**, in routes file - we define routes
consider creating file **models.py** and define such data there